ocr/static/ocr/js/rbi_table.js

```diff
@@ -191,65 +191,65 @@
         valign: 'middle',
         clickToSelect: false,
         events: window.operateEvents,
-        formatter: operateFormatter,
+        formatter: operateFormatter
       }],
-      [],
-    ],
-  });
+      []
+    ]
+  })
   $table.on('check.bs.table uncheck.bs.table check-all.bs.table uncheck-all.bs.table',
     () => {
-      $remove.prop('disabled', !$table.bootstrapTable('getSelections').length);
+      $remove.prop('disabled', !$table.bootstrapTable('getSelections').length)
 
       // save your data, here just save the current page
-      selections = getIdSelections();
+      selections = getIdSelections()
     // push or splice the selections if you want to save all data selections
-    });
+    })
   $table.on('all.bs.table', (e, name, args) => {
     /* eslint no-console:0 */
-    console.log(name, args);
-  });
+    console.log(name, args)
+  })
 }
 
-function customButtons() {
+function customButtons () {
```

Comment:
Codacy has a fix for the issue: [Unexpected space before function parentheses.](https://app.codacy.com/gh/PrynsTag/oneBarangay/pullRequest?prid=8186851)
```suggestion
function customButtons() {
```